### ISSUE TITLE
Does it still work?

### ISSUE THREAD

**Issue by xeruf** [z3ntu/MonstercatConnectHelper#2]
Since connect itself is closed, how do I use it now? I can't find any button or smth.

**Comment by z3ntu** [comment]
Yes... I'll look into it in a few days and report back about what's still possible. Also which functionality do you want/need?

**Comment by xeruf** [comment]
I recently bought Monstercat Gold and now was searching for an easy way to download everything. I thought about writing my own script but then I realised that it is pretty likely that someone already did that.

**Comment by z3ntu** [comment]
Ah okay. I don't have Gold anymore but maybe I can buy it for a month so get the extension up and running again :)

**Comment by z3ntu** [comment]
I can't promise you much as I'll probably be busy until the middle of October but I can try
